### PR TITLE
ClickHouse Operator rolling udpate logic testcase

### PR DESCRIPTION
**Pull request by roshanths**
Thanks for taking the time to contribute to `clickhouse-operator`!

Please, read carefully [instructions on how to make a Pull Request](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#intro).

This will help a lot for maintainers to adopt your Pull Request. 

## Important items to consider before making a Pull Request
Please check items PR complies to:
* [ ] All commits in the PR are squashed. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#how-to-make-pr) 
* [ ] The PR is made into dedicated `next-release` branch, **not into** `master` branch<sup>1</sup>. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#how-to-make-pr)
* [ ] The PR is signed. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#sign-your-work)

--

<sup>1</sup> If you feel your PR does not affect any Go-code or any testable functionality (for example, PR contains docs only or supplementary materials), PR can be made into `master` branch, but it has to be confirmed by project's maintainer.

### CODE REVIEW

File: tests/e2e/test_operator.py [hunk 1]
```diff
@@ -2060,6 +2062,116 @@ def test_031(self):
...
+
+
+@TestScenario
+@Name("test_032. Test rolling update logic")
+def test_032(self):
+    """Test rolling update logic."""
+
+    util.require_keeper(keeper_type=self.context.keeper_type)
+    create_table = """
+    CREATE TABLE test_local(a UInt32)
```
Comment (Slach) [inline tests/e2e/test_operator.py:2091]: you create table only in one replica
use 
```sql
CREATE TABLE test_local ON CLUSTER 'default' ...
```

File: tests/e2e/test_operator.py [hunk 2]
```diff
@@ -2163,14 +2163,13 @@ def test_032(self):
...
-
     note("Setting the thread event to true...")
     trigger_event.set()
+    note("Joining the parallel thread with main...")
     join()
 
-    kubectl.launch('delete clickhouse-test-032-client')
     kubectl.delete_chi(chi)
-
+    kubectl.launch(f"delete pod clickhouse-test-032-client", ns=kubectl.namespace, timeout=600, ok_to_fail=True)
```
Comment (Slach) [inline tests/e2e/test_operator.py:2172]: ok_to_fail=False shall be here client just a  simple pod without external dependencies and shall delete without error

**Review by Slach — COMMENTED** [review]
pls fix https://github.com/Altinity/clickhouse-operator/pull/936/files#r876509373

File: tests/e2e/test_operator.py (hunk 2 above)
Comment (roshanths) [reply tests/e2e/test_operator.py]: Ok, change is commit now

**Review by Slach — APPROVED** [review]

**Pull request merged**